executor/write.go

```diff
@@ -1915,3 +1924,17 @@ func (e *UpdateExec) Close() error {
 func (e *UpdateExec) Open(goCtx goctx.Context) error {
 	return e.SelectExec.Open(goCtx)
 }
+
+func updateTableDelta(ctx context.Context, tableInfo *model.TableInfo, row []types.Datum, delta int64, count int64) {
```

Comment:
Why not call this function in `AddRecord`, `RemoveRecord` and `UpdateRecord` functions ?

Reply:
First, to put them in these functions would require adding a new method for `context.Context`. Second, there are some places that do not need the DML update, for example when importing data, because we will auto analyze them anyway.